generator/generator.yml

```diff
@@ -1090,3 +1090,15 @@ modules:
         scale: .01
       jnxDomCurrentTxLaserBiasCurrent:
         scale: .001
+
+# Yamaha RT Series
+#
+# http://www.rtpro.yamaha.co.jp/RT/docs/mib/yamaha-private-mib.zip
+  yamaha_rt:
+    walk:
+      - yrhMemorySize
+      - yrhMemoryUtil
+      - yrhCpuUtil5sec
+      - yrhCpuUtil1min
+      - yrhCpuUtil5min
+      - yrfUpTime
```

Comment:
Might be useful to add a scale override here to convert the value from "in hundredths of a second" to seconds.